ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -151,6 +151,13 @@ objects:
           --zone "${GOOGLE_COMPUTE_ZONE}" \
           packer@"${INSTANCE_PREFIX}" \
           --command 'export KUBECONFIG=/home/packer/clusters/nested/auth/kubeconfig && /home/packer/run-tests.sh'
+        echo "scp junit logs back to test pod"
+        LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
```

Comment:
We are `set -e`, so won't this never run if  `run-tests.sh` fails?  Maybe it should live in the `teardown` container?